pkg/clusterd/disk.go

```diff
@@ -76,7 +76,12 @@ func DiscoverDevices(executor exec.Executor) ([]*sys.LocalDisk, error) {
 		}
 
 		diskType, ok := diskProps["TYPE"]
-		if !ok || (diskType != sys.SSDType && diskType != sys.CryptType && diskType != sys.DiskType && diskType != sys.PartType) {
+		if !ok || (diskType != sys.SSDType && diskType != sys.CryptType && diskType != sys.DiskType && diskType != sys.PartType && diskType != sys.LinearType) {
+			if !ok {
+				logger.Warningf("skipping device %s: diskType is empty", d)
```

Comment:
`Debugf`?

Reply:
The usage of Warningf is following the other similar checks within this function.

Does it make sense that I make all of them Debugf ?

Reply:
No, let's keep it this way then.